upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -418,6 +421,34 @@ func (c *ApplyClusterCmd) Run() error {
 	l.WorkDir = c.OutDir
 	l.ModelStore = modelStore
 
+	iamLifecycle := lifecyclePointer(fi.LifecycleSync)
+	networkLifecycle := lifecyclePointer(fi.LifecycleSync)
+	clusterLifecycle := lifecyclePointer(fi.LifecycleSync)
+
+	switch c.Phase {
```

Comment:
I would definitely like to have the capability to run both `PhaseNetwork` and `PhaseCluster` at the same time.